isis/src/base/apps/enlarge/enlarge_app.cpp

```diff
@@ -0,0 +1,93 @@
+#include "Enlarge.h"
+#include "enlarge_app.h"
+#include "IException.h"
+#include "ProcessRubberSheet.h"
+
+using namespace std;
+using namespace Isis;
+namespace Isis{
+
+  void enlarge(UserInterface &ui, Pvl *log) {
+    Cube *cube = new Cube(ui.GetFileName("FROM"), "r");
```

Comment:
You need to check for CubeAttributeInputs:
https://github.com/USGS-Astrogeology/ISIS3/blob/dev/isis/src/base/apps/cam2map/cam2map.cpp#L29